.github/workflows/functional-tests.yml

```
@@ -60,58 +173,97 @@
         working-directory: ${{ env.frontend-directory }}
-        run: |
-          npm install -g pnpm
-          pnpm i --frozen-lockfile
+        run: pnpm i --frozen-lockfile
+
+      - name: Get installed Playwright version
+        id: playwright-version
+        run: echo "PLAYWRIGHT_VERSION=$(pnpm exec playwright --version | awk -F ' ' '{print $2}')" >> $GITHUB_ENV
+
+      - name: Cache playwright binaries
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)